build.psm1

```diff
@@ -1986,6 +1998,11 @@ function New-MSIPackage
     $wixCandleExePath = Join-Path $wixToolsetBinPath "Candle.exe"
     $wixLightExePath = Join-Path $wixToolsetBinPath "Light.exe"
 
+    if ($null -eq (Get-Module packaging))
+    {
+        # Import Get-PackageSemanticVersion function from packaging module
+        Import-Module "$PSScriptRoot\tools\packaging\packaging.psd1"
```

Comment:
I was sure we were using auto-loading.

Reply:
See the example where it only imports build.psm1. We would need to have a module manifest for the build module then. Or maybe the `New-MSIPackage` should be moved into the packaging module?

Reply:
New-MSIPackage should be moved